CHANGELOG.md

```diff
@@ -6,9 +6,53 @@ Extends support of the SDK to OneView REST API version 3200 (OneView v6.30)
 - [#173] (https://github.com/HewlettPackard/oneview-python/issues/73) is this a typo https://github.com/HewlettPackard/oneview-python/blob/master/examples/server_hardware.py#L200-201
 
 ##### Features supported with the current release
```

Comment:
In first line change unreleased to released

Reply:
we are not releasing this yet. Its just the validation.

Reply:
okay